### ISSUE TITLE
Chore: Fix Broken Link to 'Contributing to Open Source' in Landing Page Footer

### ISSUE THREAD

**Issue by Sriparno08** [Sriparno08/Openpedia#346]
## Category

- [ ] Documentation
- [ ] Resource Addition
- [x] Codebase
- [ ] User Interface
- [ ] Feature Request

## Description

We need to fix the broken link to 'Contributing to Open Source' in landing page footer by replacing the current link with the following link:

> https://openpedia.netlify.app/contributing%20to%20opensource/

**Comment by Bibek478** [comment]
Please assign me this issue under jwoc24

**Comment by Sriparno08** [comment]
You're already assigned to another issue. Once your PR for it is merged, you will be considered for this issue. If anyone asks to be assigned before your PR is merged, he/she will be assigned.

**Comment by Bibek478** [comment]
ok

**Comment by thatguywhocode** [comment]
Hey @Sriparno08 , assign me this issue under JWOC

**Comment by Sriparno08** [comment]
@thatguywhocode You're assigned.

**Comment by thatguywhocode** [comment]
Hey @Sriparno08 , i created pr under JWOC.
